### ISSUE TITLE
[Bug] "alt" missing in img tags

### ISSUE THREAD

**Issue by SisiVero** [creativecommons/vocabulary#105]
## Description
the alt feature that comes with the img tag is missing in all the img tags in each html page

Why add "alt" to tags?
1. The "alt" aids users who make use of screen readers. It makes it easy for the screen reader to identify what the image is about.

2. In cases of slow network and a page takes time to load, hence images not loading, the alt text will appear beside the unloaded image giving the user a description of what the image is about.

## Expectation
The alt feature  will help users who make use of screen readers. 
It will also provide a description or an alternate text to display when an image does not load.

- [x] I would be interested in resolving this bug.

**Comment by possumbilities** [comment]
Hi and thanks for this, I believe this would largely be blocked by the efforts of #39 
Adding necessary Alt text could be folded into that Issue as it is resolved, but should likely wait until then.

Moving to `status: blocked`.
